tests/rtt_ros2_idl_tests/CMakeLists.txt

```
@@ -4,6 +4,7 @@ project(rtt_ros2_idl_tests)
 # Skip building this package if BUILD_TESTING is OFF
 option(BUILD_TESTING "Build the testing tree." ON)
 if(NOT BUILD_TESTING)
+  find_package(ament_cmake REQUIRED)
   ament_package()
   return()
 endif()
```

In this case better revert to

```suggestion
find_package(ament_cmake REQUIRED)
if(NOT BUILD_TESTING)
  ament_package()
  return()
endif()
```

as it has been before https://github.com/orocos/rtt_ros2_integration/commit/2678b269b4edaa829e3e27633c833e79e968aa9a. This commit caused this regression bug.

As an alternative we could try
```cmake
option(BUILD_TESTING "Build the testing tree." ON)
if(NOT BUILD_TESTING)
  install(CODE "")  # Install nothing. Otherwise make install would fail.
  return()
endif()
```
to not even install an empty `package.xml` and other files generated by `ament_package()`. Not sure whether that works.

---

Applied the second alternative in https://github.com/orocos/rtt_ros2_integration/pull/20/commits/50199c0eb97d57859c29deeba73e97bdfae09032.